src/OrbitVulkanLayer/SubmissionTracker.h

```diff
@@ -934,6 +920,32 @@ class SubmissionTracker : public VulkanLayerProducer::CaptureStatusListener {
     }
   }
 
+  // This method does not acquire a lock and MUST NOT be called without holding the `mutex_`.
+  void ResetCommandBufferUnsafe(VkCommandBuffer command_buffer) {
```

Comment:
"Unsafe" sounds a bit worse that it actually is. But I don't really know what a better word would be. "Unsynchronized"/"Unsynch" maybe? Or feel free to leave it as is if you don't like it.

Reply:
I've seen that in a couple of other projects, also used for missing synchronisation. The nice thing is that it actually sounds bad. So when you use it, you will look at the method, to check "why is it unsafe".

However, I don't feel strongly about this and could also use "Unsynchronized".

Reply:
Hehe that's an interesting point of view. I don't have a strong opinion either, I just wanted to mention it.

Reply:
I guess I'll leave it like this.

Reply:
> Like the other change I think it would make sense to move some details from the commit message into one (or multiple if necessary) comments, so that the explanation doesn't get lost. Looks good otherwise.

Thanks. Added two comments.